ninja-core/src/main/java/ninja/session/Session.java

```diff
@@ -30,7 +30,8 @@
     String AUTHENTICITY_KEY = "___AT";
     String ID_KEY = "___ID";
     String TIMESTAMP_KEY = "___TS";
-    
+	String EXPIRY_TIME_KEY = "___EP";
```

Comment:
really tiny... but please make sure to use spaces and not tabs in the source code...  :)